addon/components/learnergroup-tree.js

```diff
@@ -2,14 +2,18 @@ import Component from '@glimmer/component';
 import { tracked } from '@glimmer/tracking';
 import { action } from '@ember/object';
 import { filter } from 'rsvp';
-import { isEmpty } from '@ember/utils';
+import { isEmpty, isPresent } from '@ember/utils';
 import {restartableTask} from "ember-concurrency-decorators";
 
 export default class LearnergroupTree extends Component {
   @tracked isHidden = true;
   @tracked selectable = false;
   @tracked hasChildren = false;
 
+  get isRoot() {
+    return isPresent(this.args.isRoot) ? this.args.isRoot : true;
```

Comment:
Is `isPresent` needed here? Seems like this is either `true` or falsey (`false`, `undefined`).